paver.sh

```diff
@@ -552,6 +552,12 @@ detect_os() {
                 check_os_version "SUSE Linux Enterprise Server" 10 \
                     "$os_version_raw" os_version_chevah
                 OS="sles${os_version_chevah}"
+                if [ ${os_version_chevah} -eq 11 ]; then
+                    # Check for SLES 11 with OpenSSL 1.0 (aka Security Module).
+                    if [ -n $(rpm -q openssl1) ]; then
```

Comment:
We could check for `/usr/bin/openssl1`, but the RPM check is stronger. And the `rpm` binary is practically guaranteed to exist on a SLES 11 system. No strong feelings though…

Perhaps we should also emit a message in the console in SLES 11 if OpenSSL 1.0.x is not found?